linchpin/validator.py

```diff
@@ -0,0 +1,53 @@
+from cerberus import Validator
```

Comment:
I don't think this file should reside in the api root. It's likely going to land in the util namespace. What do you think of that?

Reply:
That's fine with me